ceph/rados/core_workflows.py

```diff
@@ -1772,3 +1824,67 @@ def get_ceph_pg_dump(self, pg_id: str) -> dict:
 
         log.error(f"PG {pg_id} not found in ceph pg dump output")
         raise KeyError(f"PG {pg_id} not found in ceph pg dump output")
+
+    def restart_daemon_services(self, daemon: str):
```

Comment:
should we have this module to start, stop and restart? Looks good for now,  We can have enhancements in the next PR.

Reply:
Noted, will create a backlog task